app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -17,6 +17,10 @@ def initialize(authentication_input:, jwt_token:)
       def authn_jwt_variable_id_prefix
         "#{@account}:variable:conjur/#{@authenticator_name}/#{@service_id}"
       end
+
+      def hash
+        "#{@service_id}-#{@authenticator_name}-#{@account}"
```

Comment:
i think the order should be:
`"#{@account}-#{@authenticator_name}-#{@ service_id}"`

Reply:
Are you going to remove it?

Reply:
Removed